lib/auth/grpcserver_test.go

```diff
@@ -1018,3 +1018,59 @@ func TestDeleteLastMFADevice(t *testing.T) {
 		checkErr: require.Error,
 	})
 }
+
+// TestClusterNetworkingConfigOriginDynamic tests setting ClusterNetworkingConfig
+// via gRPC results in configuration with OriginDynamic being stored.
+func TestClusterNetworkingConfigOriginDynamic(t *testing.T) {
+	ctx := context.Background()
```

Comment:
```suggestion
        t.Parallel()
	ctx := context.Background()
```
Can this run parallel to other tests?